corehq/apps/users/util.py

```diff
@@ -317,7 +317,7 @@ def _last_sync_needs_update(last_sync, sync_datetime):
 def log_user_change(by_domain, for_domain, couch_user, changed_by_user, changed_via=None,
                     change_messages=None, fields_changed=None, action=None,
                     by_domain_required_for_log=True, for_domain_required_for_log=True,
-                    bulk_upload_record_id=None):
+                    bulk_upload_record_id=None, group_changes_only=False):
```

Comment:
Can you share why this change was needed?
I believe you should be able to log with just the change_messages and passing blank fields changed.

Reply:
You're right about that. Thanks, making the change.